docs/source/write_test.rst

```diff
@@ -10,7 +10,6 @@ The file structure of a basic DICE project likes::
     project_root
     |-- oracles
     |   `-- pyramid.yaml
-    |-- item.py
```

Comment:
The change is not just removing item.py, but moving it into utils folder. Please fix this according to the current example structure.